### DISCUSSION TITLE
Error with copying Toast component stories to website

### DISCUSSION THREAD

**Discussion by realEddieZhong** [chakra-ui/ark#2085] [General]
### Description

There is no sample code for Vue.

### Link to Reproduction (or Detailed Explanation)

https://ark-ui.com/docs/components/toast

### Steps to Reproduce

https://ark-ui.com/docs/components/toast

### Ark UI Version

0.8.0

### Framework

- [ ] React
- [ ] Solid
- [X] Vue

**Comment by realEddieZhong** [comment]
Now:
```
<script setup lang="ts">
null
</script>

<template><button @click="createBasicToast">Toast</button> <BasicToaster /></template>
```

**Comment by Omikorin** [comment]
It you need examples now, please look at [the source code](https://github.com/chakra-ui/ark/blob/main/packages%2Fframeworks%2Fvue%2Fsrc%2Ftoast%2Ftoast.story.vue).

Unfortunately, because of the implementation, our current script failed to copy stories to the website.

**Comment by realEddieZhong** [comment]
That's ok, thanks! I thought you forgot, haha

**Comment by Omikorin** [comment]
I'll reopen this issue as we need to address the problem later.
